File: app/src/main/java/com/drivingschoolrwandaapp/models/entities/UserActivity.java
package com.drivingschoolrwandaapp.models.entities;

import com.google.gson.annotations.SerializedName;

import java.io.Serializable;

public class UserActivity implements Serializable {
    @SerializedName("id")
    private int id;

    @SerializedName("activityType")
    private String activityType;

    @SerializedName("description")
    private String description;

    @SerializedName("createdAt")
    private String createdAt;

    // Getters and Setters
    public int getId() { return id; }
    public String getActivityType() { return activityType; }
    public String getDescription() { return description; }
    public String getCreatedAt() { return createdAt; }
}

File: app/src/main/java/com/drivingschoolrwandaapp/models/entities/UserActivity.kt
package com.drivingschoolrwandaapp.models.entities

import com.google.gson.annotations.SerializedName
import java.io.Serializable

data class UserActivity(
    @SerializedName("id") var id: Int = 0,
    @SerializedName("activityType") var activityType: String? = null,
    @SerializedName("description") var description: String? = null,
    @SerializedName("createdAt") var createdAt: String? = null
) : Serializable
